grouch/ui.py

```diff
@@ -98,7 +98,7 @@ def courses(args, store):
         d = course['description']
         if d is not None:
           for line in wrapper.wrap(d):
-            print line
+            print (line)
```

Comment:
The space before the paren is unconventional - I'd prefer `print(line)` to `print (line)`.